Library/Homebrew/tap.rb

```diff
@@ -100,7 +100,7 @@ def initialize(user, repo)
     @user = user
     @repo = repo
     @name = "#{@user}/#{@repo}".downcase
-    @full_name = "#{@user}/homebrew-#{@repo}"
+    @full_name = (@repo == "brew") ? "#{user}/#{repo}" : "#{@user}/homebrew-#{@repo}"
```

Comment:
I don't really like this.

Reply:
Yeah I don't think this belongs here really.

Reply:
Yeah, me neither, it was just what I had on hand at the time.